django/db/migrations/state.py

```diff
@@ -188,14 +211,20 @@ def add_field(self, app_label, model_name, name, field, preserve_default):
         else:
             field = field
         self.models[app_label, model_name].fields[name] = field
+        if self._relations is not None:
+            model_key = (app_label, model_name)
```

Comment:
Similarly, you could define `model_key` before the `self.models[app_label, model_name].fields[name] = field` line above, which would let you use it there as well.